### ISSUE TITLE
ratings per individual user

### ISSUE THREAD

**Issue by LinguList** [concepticon/norare-data#166]
We have situations like the list by Jackson et al. 2019 where we have also individual non-aggregated ratings of anonymized users. The question we should ask ourselves is: do we want to be able to handle them as well? And if so, how? We could of course list the original data in some cleaned TSV, and do the aggregation automatically. Or we could have a large TSV with many columns, which reflect individual users' ratings...Or we could add a JSON file to the repository that contains the user ratings and could be loaded via norare's access to the data path.

**Comment by AnnikaTjuka** [comment]
Good point. I can only recall one other study that included non-aggregate ratings in addition to aggregate ratings. So a ratio of 1-2 for approx. 100 different data sets. If this is not going to become more common, it is probably best to use the data path.

**Comment by AnnikaTjuka** [comment]
Sounds good! I think we talked about releasing Concepticon 3.0 / NoRaRe 1.0 in April.

**Comment by LinguList** [comment]
That may in fact be feasible! We would only need to find a solution to 
work on norare and concepticon in the same repository (maybe with git 
subdirectories). Let us try to schedule a meeting with @xrotwang early 
april on this.
